code/_onclick/hud/radial.dm

```diff
@@ -245,24 +249,15 @@ GLOBAL_LIST_EMPTY(radial_menus)
 		update_screen_objects()
 
 /datum/radial_menu/proc/show_to(mob/M)
-	if(current_user)
-		hide()
 	if(!M.client || !anchor)
 		return
-	current_user = M.client
 	//Blank
-	menu_holder = image(icon = 'icons/effects/effects.dmi', loc = anchor, icon_state = "nothing", layer = ABOVE_HUD_LAYER)
-	menu_holder.appearance_flags |= KEEP_APART
-	menu_holder.vis_contents += elements + close_button
-	current_user.images += menu_holder
-
-/datum/radial_menu/proc/hide()
-	if(current_user)
-		current_user.images -= menu_holder
+	close_button.screen_loc = "CENTER:[pixel_x_difference], CENTER:[pixel_y_difference]"
```

Comment:
Weirdly, for default by screenloc we don't have spaces after the commas
```suggestion
	close_button.screen_loc = "CENTER:[pixel_x_difference],CENTER:[pixel_y_difference]"
```

Reply:
Dupe code removed, reviews addressed